### PR TITLE
Refactored log statements for OCSInitialization and PersistentVolume controllers

### PR DESCRIPTION
**Pull request by priyanka19-98**
Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

File: controllers/persistentvolume/reconcile.go [hunk 1]
```diff
@@ -28,19 +28,19 @@ func (r *PersistentVolumeReconciler) Reconcile(ctx context.Context, request reco
...
+			r.Log.Info("PersistentVolume not found.", "PersistentVolume", request.Name)
 			return reconcile.Result{}, nil
 		}
 		return reconcile.Result{}, err
 	}
 
 	// Check GetDeletionTimestamp to determine if the object is under deletion
 	if !pv.GetDeletionTimestamp().IsZero() {
-		r.Log.Info("Object is terminated, skipping reconciliation")
+		r.Log.Info("Object is terminated, skipping reconciliation.", "PersistentVolume", pv.Name)
```
[the remark this answers is not in the document]
Comment (priyanka19-98) [reply controllers/persistentvolume/reconcile.go]: done

**Review by umangachapagain — COMMENTED** [review]
Can you add go.mod and go.sum changes in a separate commit?
Thanks.

**Comment by priyanka19-98** [comment]
> Can you add go.mod and go.sum changes in a separate commit?
> Thanks.

done

**Comment by priyanka19-98** [comment]
> > Can you add go.mod and go.sum changes in a separate commit?
> > Thanks.
> 
> done 
Not sure, why the 1st commit still shows go.mod and vendor/modules.txt changes.

**Review by umangachapagain — APPROVED** [review]
Thanks for the refactor.

/lgtm

**Pull request merged**